rpc/v2/errors.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2015 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+
+package v2
+
+import "fmt"
+
+// request is for an unknown service
+type unknownServiceError struct {
+	service string
+	method  string
+}
+
+func (e *unknownServiceError) Code() int {
+	return -32601
+}
+
+func (e *unknownServiceError) Error() string {
+	return fmt.Sprintf("Unknown service %s%s%s", e.service, serviceMethodSeparator, e.method)
```

Comment:
We can also stay closer to the spec by setting the error string to something like `"%s method does not exist / is not available in service %s"`